src/plasoscaffolder/bll/mappings/mapping_helper.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""Helper methods for mapping"""
+import os
+
+from jinja2 import Environment
+from jinja2 import FileSystemLoader
+
+
+def _get_template_path() -> os.path:
+  """
+  the path to the template file
+  :param file_name: the name of the file
+  :return: the template file path
+  """
+  path = os.path.dirname(os.path.abspath(__file__))
+  head, tail = os.path.split(path)
```

Comment:
Maybe use different variable names to indicate their purpose/contents:
dirname, basename = os.path.split(path)